pkg/porter/build_test.go

```diff
@@ -39,6 +40,36 @@ func TestPorter_buildDockerfile(t *testing.T) {
 	assert.Equal(t, wantlines, gotlines)
 }
 
+func TestPorter_buildDockerfile_output(t *testing.T) {
+	buffer := bytes.NewBuffer([]byte{})
```

Comment:
I would like to show you a special function that I built into TestPorter that assists with collecting output during a test.

You can remove lines 44 and 46. Then on line 70, change `buffer.String` to `p.TestConfig.TestContext.GetOutput()`. 

The TestPorter struct does the same thing that you did in the test (make a byte buffer) and then exposes it as a string with `GetOputput()`.

Reply:
That makes sense to me! Just updated the branch to reflect those changes.